docs/how-to/revert-from-rt-kernel.md

```diff
@@ -0,0 +1,121 @@
+# How to revert from real-time kernel to generic kernel
+
+Reverting from the real-time kernel to the generic kernel requires manual intervention.
+This guide provides step-by-step instructions for properly disabling the real-time kernel and restoring the system to a generic kernel configuration.
+It applies to both **GRUB-based systems** and **Raspberry Pi** devices running Ubuntu Server or Ubuntu Desktop.
+
+```{warning}
+This guide does not apply to Ubuntu Core systems
+```
+(ensure-the-presence-of-other-kernels)=
+## Ensure the presence of other kernels
+
+Before removing the real-time kernel, check whether other kernel versions are already installed.
+Search for installed kernels excluding the real-time ones:
+
+```{terminal}
+    :input: apt list --installed | grep -v realtime | grep linux-image
+    :user: ubuntu
+    :host: ubuntu
+
+WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
+
+linux-image-6.11.0-29-generic/noble-updates,noble-security,now 6.11.0-29.29~24.04.1 amd64 [installed,automatic]
+linux-image-generic-hwe-24.04/noble-updates,noble-security,now 6.11.0-29.29~24.04.1 amd64 [installed,automatic]
+```
+
+In case that other kernels are already installed, as shown in the example above, proceed to [disabling real-time Pro service](#disable-rt-on-pro).
+If the system has no kernels installed other than the real-time ones, you must install a generic kernel first:
```

Comment:
```suggestion
In case non-realtime kernels are already installed, proceed to [disabling real-time Pro service](#disable-rt-on-pro).
Otherwise, install a generic kernel first:
```